### DISCUSSION TITLE
Can I format data as a link?

### DISCUSSION THREAD

**Discussion by andrewducker** [EvotecIT/PSWriteHTML#459] [Q&A] [answered]
I expected that if I passed in `[MyLink](http://mysite.com/mylink)` as the contents of a column in a Dashimo-style table then I'd get a link appearing in the column.
Instead I get that as text.
Is there any way to make it a link?
(Basically I'm listing a bunch of data, one column of which is a link to the project the data is about, and I'd like that data to be a hyperlink to it.)

**Comment by PrzemyslawKlys** [comment]
Are you using `new-htmltable -InvokeHTMLTags` with that? I do support markdown, but probably not in Tables directly. You would need to use 

```powershell
[PSCustomObject] @{
   xxx = New-HTMLText -Text "xxx [MD](xxx)" 
}
```

for it to work. Alternatively you can use real HTML directly, but you need the InvokeTags for it to be activated.

**Comment by andrewducker** [comment]
I've just tried New-HtmlText and that didn't work - 

```
$someData = @()
$someData += [pscustomobject]@{
	Link = New-HtmlText -Text "[My link](http://microsoft.com)"
	MyData = 1
}
$someData += [pscustomobject]@{
	Link = New-HtmlText -Text "[my other link](http://google.com)"
	MyData = 2
}


Dashboard -Name "Test" -FilePath d:\zzzz\testdashboard.html {
	Tab -Name "Details"{
		Table -Datatable $someData 
	}
}
```

I get:
![image](https://github.com/user-attachments/assets/1ffddd99-0bd3-4daf-ab01-45c30011489d)

Where do I tell it I want to use HTML Tags?

Or do I need to use a different approach?

**Reply by PrzemyslawKlys** [reply, marked as answer]
`Table -InvokeHTMLTags -DataTable $someData` in your example

**Reply by andrewducker** [reply]
Perfect, thank you so much!
